README.md

```diff
@@ -0,0 +1,11 @@
+
+# robot_model
+
+
+### This is a fork repository of [ros/robot_model](https://github.com/ros/robot_model).
```

Comment:
The proposed merge is not a fork... I think you should remove this readme all together from the pull request.